1603. Design Parking System

```diff
@@ -0,0 +1,27 @@
+class ParkingSystem(object):
```

Comment:
this was obviously just an easy, but I think there are good interview problems for related to parking systems u should google that are on the medium/harder level for both coding and design.

Reply:
will take a look